v2/futures/client.go

```diff
@@ -525,7 +525,10 @@ func (c *Client) NewGetBalanceService() *GetBalanceService {
 	return &GetBalanceService{c: c}
 }
 
-// NewGetPositionRiskService init getting position risk service
+func (c *Client) NewGetPositionRiskV2Service() *GetPositionRiskV2Service {
+	return &GetPositionRiskV2Service{c: c}
+}
+
```

Comment:
Is there any reason to keep this V2 service? In my opinion, we only need the latest version of each service.

Reply:
Binance still provides this interface, so I have kept it. Of course, I can also keep only the V3 version

Reply:
> Binance still provides this interface, so I have kept it. Of course, I can also keep only the V3 version @adshao

OK, that's fine, we can keep it as long as binance provides both v2/v3 interfaces.